esmvaltool/preprocessor/_mask.py

```diff
@@ -17,20 +19,81 @@
 logger = logging.getLogger(__name__)
 
 
-def mask_landocean(cube, land_mask=None, ocean_mask=None):
-    """Mask land or ocean"""
-    if land_mask or ocean_mask:
-        raise NotImplementedError
-
+def _get_fx_mask(fx_data, fx_option):
+    """Build a 50 percent land or ocean mask"""
+    inmask = np.zeros_like(fx_data, bool)
+    if fx_option == 'land':
+        # Mask land out
+        inmask[fx_data <= 50.] = False
```

Comment:
the line above can be removed, as you're already starting with an array full of zeros.